formbricks/formbricks
# feat/allow question id change

## What does this PR do?



Adds the option to edit the question id of questions added to running surveys before getting saved




https://github.com/formbricks/formbricks/assets/42251569/15a1b032-8e8d-4bc8-8d20-5dc69d038ed0


## Type of change




## How should this be tested?



- Add a new question to a running survey and see if id is editable
- Check if question ids in drafted survey are editable

## Checklist
